### PR TITLE
changes how parameters are received

### LINKED ISSUE #4: Fix import from holidays
Currently the import is this way 
`const holidays = require('./holidays/2017')`

we need to change it so it imports the file from the current year (or for future implementations, past years)

I'm going to work on this and send the PR

### PR DESCRIPTION
**Pull request by vcapretz**
lots of changes were required, all tests pass (even new ones)

*the is-holiday project needs changes after this PR is merged*

closes #4

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,30 +1,36 @@
 'use strict'
 
-const holidays = require('./holidays/2017')
+const holidays = require('./holidays/')
+
+const holiday = obj => {
```
Comment (bukinoshita) [inline index.js]: If I run:
```js
holiday()
```

it returns me this error:
```bash
/Users/buuh7921/github/holiday/index.js:7
  const {year = date.getFullYear(), month, day, country = 'us'} = obj
                                                                  ^
TypeError: Cannot match against 'undefined' or 'null'.
```

You probably want to use it like this:
```js
const holiday = (opts = {}) => {
  ...
}
```

Ps: change `obj` to `opts`

Comment (bukinoshita) [reply index.js]: Also create a test for that, running the function without any arguments should return the entire year

**Review by bukinoshita — CHANGES_REQUESTED** [review]
It's looking great! Some minor changes and if possible to update the README as well, would be great since the API has changed.

**Comment by vcapretz** [comment]
@bukinoshita I also changed the output for the entire year because it was just an array and we could know the day and month of each holiday (we will need it to `is-holiday`)

**Review by bukinoshita — APPROVED** [review]

**Comment by bukinoshita** [comment]
Looks great @vcapretz 🎉

**Pull request merged**